model/src/w3sdb1md.F90

```diff
@@ -235,8 +236,11 @@ SUBROUTINE W3SDB1 (IX, A, DEPTH, EMEAN, FMEAN, WNMEAN, CG, LBREAK, S, D )
     S = 0.
     D = 0.
 
-    THR = DBLE(1.E-15)
-    IF (SUM(A) .LT. THR) RETURN
+    IF (EMEAN .LT. TINY(1.d0)) THEN
+      S = 0 
```

Comment:
Are `S` and `D` not zeroed above? Why zero again here?

Reply:
yes, this was some leftover i cleaned it now, thanks @ukmo-ccbunney